src/modules/features/ModuleSetupObsidian.ts

```diff
@@ -139,11 +139,11 @@ export class ModuleSetupObsidian extends AbstractObsidianModule implements IObsi
                     newSettingW.encryptedPassphrase = "";
                     newSettingW.encryptedCouchDBConnection = "";
                     newSettingW.additionalSuffixOfDatabaseName = `${"appId" in this.app ? this.app.appId : ""}`;
-                    const setupJustImport = "Just import setting";
-                    const setupAsNew = "Set it up as secondary or subsequent device";
-                    const setupAsMerge = "Secondary device but try keeping local changes";
-                    const setupAgain = "Reconfigure and reconstitute the data";
-                    const setupManually = "Leave everything to me";
+                    const setupJustImport = "Don't sync anything, just apply the settings.";
+                    const setupAsNew = "This is a new client - sync everything from the remote server.";
+                    const setupAsMerge = "This is an existing client - merge existing files with the server.";
+                    const setupAgain = "This is a new server - reset remote server and use local files.";
```

Comment:
"This is a new server":  Can you make it a bit more nuanced, like ‘this client creates new server data’?
`This is for a new or broken server`? Is it too long?

Reply:
I agree, my wording was a little simple there - I was trying to keep them as short as possible to make it easy to read at a glance, even if it left out some information.
I've changed it to `Initialise new server data - ideal for new or broken servers.`
There's probably a even better way of wording it, but this seems much better for now :)